package.json

```diff
@@ -90,7 +89,7 @@
     },
     "lint-staged": {
         "*.{js,css,scss}": "prettier --write",
-        "*.js": "eslint frontend/src"
+        "*.js": "yarn run eslint"
```

Comment:
I'm not sure I get this change together with removing the "eslint" script:

```
(env) Mariuss-MacBook-Pro ~/P/P/posthog [add-es-lint] (ruby 2.6.3p62) $ yarn run eslint
yarn run v1.21.1
error Command "eslint" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

![image](https://user-images.githubusercontent.com/53387/81805265-d39a1400-951a-11ea-85cf-f7dfb06c83bb.png)

Reply:
Hm weird, that did work for me. I have fixed this though.